test/SlowTests/Issues/RavenDB-21535.cs

```diff
@@ -33,8 +36,8 @@ public void KnownIssuerCert_CanNotAccess_WithoutSAN()
                 }
             });
 
-            var result = server.AuthenticateConnectionCertificate(client, null);
-            Assert.Equal(RavenServer.AuthenticationStatus.UnfamiliarCertificate, result.Status);
+            var result = server.AuthenticateConnectionCertificate(client, null, log);
+            Assert.True(RavenServer.AuthenticationStatus.UnfamiliarCertificate == result.Status, log.ToString());
```

Comment:
```suggestion
            Assert.True(RavenServer.AuthenticationStatus.UnfamiliarCertificate == result.Status, $"Expected: {RavenServer.AuthenticationStatus.UnfamiliarCertificate} but got: {result.Status}. Log: {log}");
```

I think you should add `result.Status` to see what actually got returned

Reply:
In all places like that of course